pkg/util/util_test.go

```diff
@@ -17,51 +17,64 @@ limitations under the License.
 package util
 
 import (
+	"fmt"
+	"math/rand"
 	"net"
 	"os"
 	"path/filepath"
 	"runtime"
 	"testing"
+	"time"
 
 	"k8s.io/klog"
 )
 
-var socketPath = "reg.sock"
+var socketFileName = "reg.sock"
+
+func getRandomTestDir() string {
```

Comment:
How about using this helper? https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/csi/csi_plugin_test.go#L45

Reply:
Thank you, @msau42 . Used the helper function and rebased the PR. PTAL.